compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -356,6 +356,16 @@ public void loadPredeclaredModules() {
         modules.put(Names.ERROR, this.langErrorModuleSymbol);
         modules.put(Names.OBJECT, this.langObjectModuleSymbol);
         modules.put(Names.XML, this.langXmlModuleSymbol);
+        modules.put(Names.INT, this.langIntModuleSymbol);
+        modules.put(Names.FLOAT, this.langFloatModuleSymbol);
+        modules.put(Names.DECIMAL, this.langDecimalModuleSymbol);
+        modules.put(Names.BOOLEAN, this.langBooleanModuleSymbol);
+        modules.put(Names.FUTURE, this.langFutureModuleSymbol);
+        modules.put(Names.MAP, this.langMapModuleSymbol);
+        modules.put(Names.STREAM, this.langStreamModuleSymbol);
+        modules.put(Names.STRING, this.langStringModuleSymbol);
+        modules.put(Names.TABLE, this.langTableModuleSymbol);
+        modules.put(Names.TYPEDESC, this.langTypedescModuleSymbol);
```

Comment:
Shall we order these alphabetically and have the same order as the spec (https://github.com/ballerina-platform/ballerina-spec/commit/158691e90bcc05c2f32137688d7669072fa911c0)? O:)

Reply:
Done